### PR TITLE
add scoop installed path

### PR DESCRIPTION
**Pull request by jjw24**
portable version of Everything installed by Scoop located at C:\Users\{username}\scoop\apps\everything\current\Everything.exe. Consider this as installed.

prompt the user to select a location or automatically install

### CODE REVIEW

**Comment by taooceros** [comment]
image
I think @kubalav has also encountered the same issue.

**Comment by taooceros** [comment]
Should we allow picking everything path before prompting them to install it?

**Comment by jjw24** [comment]
Yeah good idea, I will add that in

**Comment by jjw24** [comment]
@taooceros  all done

**Comment by taooceros** [comment]
We should also store the path to setting, so that user won't need to reenter it every time opening flow.

File: Main.cs [hunk 1]
```diff
@@ -195,24 +195,40 @@ private List<ContextMenu> GetDefaultContextMenu()
...
+                    System.Windows.Forms.MessageBox.Show(
                         string.Format(context.API.GetTranslation("flowlauncher_plugin_everything_installing_select"), Environment.NewLine),
                                 context.API.GetTranslation("flowlauncher_plugin_everything_installing_title"),
-                                System.Windows.Forms.MessageBoxButtons.YesNo) == System.Windows.Forms.DialogResult.Yes
-                    && string.IsNullOrEmpty(installedLocation))
+                                System.Windows.Forms.MessageBoxButtons.YesNo) == System.Windows.Forms.DialogResult.Yes)
                 {
-                    var dlg = new System.Windows.Forms.OpenFileDialog
+                    // Solves single thread apartment (STA) mode requirement error when using OpenFileDialog
+                    Thread t = new Thread(() =>
```
Comment (taooceros) [inline Main.cs:215]: It is possible to use Task? Or have to use Thread?

**Review by taooceros — APPROVED** [review]
Nice Work!

**Pull request merged**